code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -287,14 +285,13 @@ GLOBAL_LIST_INIT(sinew_recipes, list (
 	if(istype(target, /obj/mecha/working/ripley))
 		var/obj/mecha/working/ripley/D = target
 		if(D.drake_hides < DRAKE_HIDES_COVERED_FULL && !D.hides && !D.plates)
-			if(!use(1))
+			if(!use(3))
```

Comment:
Because i fucked up and was though drake leaves 4 ash drake hides, consume 3 of them prevents crafting any armor for yurself, as intended. But later there is 10 of them, which allowed to make 3 dragon ripley or darogn + armor, which is not intended too. I m glad to see one dragon ripley, but not 3 of them after 1 killing. This is too rude.